packages/nimble-components/src/checkbox/index.ts

```diff
@@ -16,7 +17,23 @@ declare global {
 /**
  * A nimble-styled checkbox control.
  */
-export class Checkbox extends FoundationCheckbox {}
+export class Checkbox extends FoundationCheckbox {
+    /**
+     * @public
+     * @remarks
+     * HTML Attribute: tabindex
+     */
+    @attr({ attribute: 'tabindex', converter: nullableNumberConverter })
+    public override tabIndex!: number;
+
+    /**
+     * @internal
+     */
+    public get resolvedTabIndex(): string | undefined {
```

Comment:
because this is resulting in the attribute format I think we should model this on the attribute name

```suggestion
    public get resolvedTabindex(): string | undefined {
```